src/main/java/com/checkmarx/flow/utils/ScanUtils.java

```diff
@@ -157,43 +157,7 @@ public static ScanRequest overrideMap(ScanRequest request, FlowOverride override
         BugTracker bt = request.getBugTracker();
         /*Override only applicable to Simple JIRA bug*/
         if(request.getBugTracker().getType().equals(BugTracker.Type.JIRA) && override.getJira()!=null) {
-            FlowOverride.Jira jira = override.getJira();
-            if(!ScanUtils.empty(jira.getAssignee())) {
-                bt.setAssignee(jira.getAssignee());
-            }//if empty value override with null
-            if(jira.getAssignee() != null && jira.getAssignee().isEmpty()) {
-                bt.setAssignee(null);
-            }
-            if(!ScanUtils.empty(jira.getProject())) {
-                bt.setProjectKey(jira.getProject());
-            }
-            if(!ScanUtils.empty(jira.getIssueType())) {
-                bt.setIssueType(jira.getIssueType());
-            }
-            if(!ScanUtils.empty(jira.getOpenedStatus())) {
-                bt.setOpenStatus(jira.getOpenedStatus());
-            }
-            if(!ScanUtils.empty(jira.getClosedStatus())) {
-                bt.setClosedStatus(jira.getClosedStatus());
-            }
-            if(!ScanUtils.empty(jira.getOpenTransition())) {
-                bt.setOpenTransition(jira.getOpenTransition());
-            }
-            if(!ScanUtils.empty(jira.getCloseTransition())) {
-                bt.setCloseTransition(jira.getCloseTransition());
-            }
-            if(!ScanUtils.empty(jira.getCloseTransitionField())) {
-                bt.setCloseTransitionField(jira.getCloseTransitionField());
-            }
-            if(!ScanUtils.empty(jira.getCloseTransitionValue())) {
-                bt.setCloseTransitionValue(jira.getCloseTransitionValue());
-            }
-            if(jira.getFields()!=null) { //if empty, assume no fields
-                bt.setFields(jira.getFields());
-            }
-            if(jira.getPriorities() != null && !jira.getPriorities().isEmpty()) {
-                bt.setPriorities(jira.getPriorities());
-            }
+            overrideJiraBugProperties(override, bt);
         }
         request.setBugTracker(bt);
```

Comment:
There are 2 very similar pieces of code starting at lines 280 and 162 (request.setBugTracker(bt)). Is it possible to extract them into a single method?